test/new/db/io/many

```diff
@@ -1,12 +1,11 @@
-# TODO: re-enable when r2r.js and v are dead
-#NAME=multiple files opened
-#FILE=<<EOF
-#-
-#-
-#-
-#EOF
-#CMDS=o~?
-#EXPECT=<<EOF
-#3
-#EOF
-#RUN
+NAME=multiple files opened
+FILE=<<EOF
+-
```

Comment:
and this?

Reply:
ok f9b988a07b1d8408d1ff1f837d7fedd726d41a5e